src/maheshKJun20/ArrayIntersection.java

```diff
@@ -0,0 +1,68 @@
+package maheshKJun20;
+
+public class ArrayIntersection {
+	
+	int count=0;
+	void findArrayIntersection(double[] arr1, double[] arr2) {
+		double[] intersectionOfArray = new double[arr1.length];
+		int counter =0;
+		for(int index=0;index<arr1.length;index++) {
+			for(int innerIndex=0;innerIndex<arr2.length;innerIndex++) {
+				if(arr2[innerIndex] == arr1[index]) {
+					intersectionOfArray[counter] = arr1[index];
+					counter++;
+				}
```

Comment:
Here after counter++, you can put break as you already get the index value. why go and check further elements and update second array value to any other value so if same element from array again come and comparing it will not comes into if condition as we already process same element.

![image](https://user-images.githubusercontent.com/58640315/86427867-fcfc4280-bd08-11ea-946d-929d3f08573c.png)

Reply:
this change save your findUniqueValueArray() method calling n comparing part.

Reply:
Thanks Harsh for the changes Suggested. I have updated and commited the code as oer above suggested changes.